src/ndv/views/_qt/_array_view.py

```diff
@@ -135,27 +142,30 @@ def __init__(self) -> None:
         super().__init__()
         self._qwidget = _QLUTWidget()
         # TODO: use emit_fast
-        self._qwidget.visible.toggled.connect(self.visibilityChanged.emit)
-        self._qwidget.cmap.currentColormapChanged.connect(self.cmapChanged.emit)
-        self._qwidget.clims.valueChanged.connect(self.climsChanged.emit)
-        self._qwidget.auto_clim.toggled.connect(self.autoscaleChanged.emit)
+        self._qwidget.visible.toggled.connect(self._on_q_visibility_changed)
+        self._qwidget.cmap.currentColormapChanged.connect(self._on_q_cmap_changed)
+        self._qwidget.clims.valueChanged.connect(self._on_q_clims_changed)
+        self._qwidget.auto_clim.toggled.connect(self._on_q_auto_changed)
 
     def frontend_widget(self) -> QWidget:
         return self._qwidget
 
     def set_channel_name(self, name: str) -> None:
         self._qwidget.visible.setText(name)
 
-    def set_auto_scale(self, auto: bool) -> None:
-        self._qwidget.auto_clim.setChecked(auto)
+    def set_clim_policy(self, policy: ClimPolicy) -> None:
+        with signals_blocked(self._qwidget.auto_clim):
+            self._qwidget.auto_clim.setChecked(not policy.is_manual)
 
     def set_colormap(self, cmap: cmap.Colormap) -> None:
-        self._qwidget.cmap.setCurrentColormap(cmap)
+        with signals_blocked(self._qwidget.cmap):
+            self._qwidget.cmap.setCurrentColormap(cmap)
```

Comment:
```suggestion
        self._qwidget.cmap.setCurrentColormap(cmap)
```

(don't think this one needs it)

Reply:
Yeah - it's not the only one. Cleaned up in [2964408](https://github.com/pyapp-kit/ndv/pull/87/commits/2964408fb94941b5b07a385182b42e27a2f5d312)